components/code-editor/README.md

```diff
@@ -0,0 +1,56 @@
+CodeEditor
+=======
+
+CodeEditor is a React component that provides the user with a code editor
+that has syntax highliting and linting.
```

Comment:
`s/highliting/highlighting/`

Reply:
Oops. Fixed in 87af208e500b0dbde3bcb68b0b846a1379692267.